charts/stable/leaf2mqtt/values.yaml

```diff
@@ -32,9 +32,6 @@ env: {}
   # CHARGING_UPDATE_INTERVAL_MINUTES:
   # LOG_LEVEL:
 
-service:
```

Comment:
Is there a UI/similar service that needs to be exposed? If so, a port number will need to be specified.

Reply:
No Service / listeners, only outbound communication.